cryptofeed/exchange/ftx.py

```diff
@@ -324,6 +324,45 @@ async def _fill(self, msg: dict, timestamp: float):
                             timestamp=float(timestamp_normalize(self.id, fill['time'])),
                             receipt_timestamp=timestamp)
 
+    async def _order(self, msg: dict, timestamp: float):
+        """
+        example message:
+        {
+            "channel": "orders",
+            "data": {
+                "id": 24852229,
+                "clientId": null,
+                "market": "XRP-PERP",
+                "type": "limit",
+                "side": "buy",
+                "size": 42353.0,
+                "price": 0.2977,
+                "reduceOnly": false,
+                "ioc": false,
+                "postOnly": false,
+                "status": "closed",
+                "filledSize": 0.0,
+                "remainingSize": 0.0,
+                "avgFillPrice": 0.2978
+            },
+            "type": "update"
+        }
+        """
+        order = msg['data']
+        await self.callback(ORDER_INFO, feed=self.id,
+                            symbol=self.exchange_symbol_to_std_symbol(order['market']),
+                            status=order['status'],
+                            order_id=order['id'],
+                            side=BUY if order['side'].lower() == 'buy' else SELL,
+                            order_type=order['type'],
+                            avg_fill_price=Decimal(order['avgFillPrice']) if order['avgFillPrice'] else None,
+                            filled_size=Decimal(order['filledSize']),
+                            remaining_size=Decimal(order['remainingSize']),
+                            amount=Decimal(order['size']),
+                            timestamp=time(),
```

Comment:
if the timestamp is not provided by the exchange, use the receipt timestamp. The case where timestamp == receipt timestamp means that the exchange did not provide any timestamp information

Reply:
Nice - this is a good approach! Will keep it in mind